src/Sheet.php

```diff
@@ -297,7 +299,11 @@ public function import($import, int $startRow = 1)
                         try {
                             app(RowValidator::class)->validate($toValidate, $import);
                             $import->onRow($sheetRow);
-                        } catch (RowSkippedException $e) {
+                        } catch (RowSkippedException) {
+                        } catch (Throwable $e) {
```

Comment:
should this be placed lower, not it only works when it has a WithValidation concern right?